src/utils/fetch.ts

```diff
@@ -1,38 +0,0 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-import fetch, { HeadersInit } from 'node-fetch';
-import config from './config.json';
-
-interface RequestOptions {
-    // @ts-ignore
-    body?: any;
-    headers?: HeadersInit;
-    method: string;
-    url: string;
-}
-
-const instance: (options: RequestOptions) => Promise<any> = async ({
-    body,
-    headers = {},
-    method,
-    url,
-}: RequestOptions) => {
-    const requestHeaders = {
-        ...headers,
-        'Passage-Version': config.version,
```

Comment:
I don't think we're including this header `Passage-Version` in the calls were making through the generated API code. This is an important field because we are using it to keep track of how many users are using which versions of our SDKs (and thus when we can kill deprecated features). 

I think you can add it to the `new Configuation` call in the functions where like the bearer token is added if I understand the generated code correctly.